src/rail/estimation/algos/deepdisc.py

```diff
@@ -143,11 +145,41 @@ class DeepDiscInformer(CatInformer):
     def __init__(self, args, comm=None):
         CatInformer.__init__(self, args, comm=comm)
 
+    def inform(self, training_data, metadata):
```

Comment:
@grantmerz mentioned that it would be ideal to have this defined as `def inform(self, training_data, training_metadata, testing_data, testing_metadata)`.  We should update the definition to accommodate that. 

Likewise, we should check for places where we use the 'input' tag for the DataStore (like on like 149). Since we'll be requiring a lot of different "inputs".

Reply:
I'd like to use "eval_data/eval_metadata" names in place of testing